tool/tsh/app.go

```diff
@@ -59,12 +70,23 @@ func onAppLogin(cf *CLIConf) error {
 			SessionID:   ws.GetName(),
 			PublicAddr:  app.GetPublicAddr(),
 			ClusterName: tc.SiteName,
+			AWSRoleARN:  arn,
 		},
 		AccessRequests: profile.ActiveRequests.AccessRequests,
 	})
 	if err != nil {
 		return trace.Wrap(err)
 	}
+
+	// Save information about the current AWS CLI access app. The CurrentAWSCLIApp is used during tsh aws call
+	// to load proper client app credentials.
+	tc.CurrentAWSCLIApp = app.GetName()
+	if err := tc.SaveProfile(cf.HomePath, true); err != nil {
+		return trace.Wrap(err)
+	}
+	if app.IsAWSConsole() {
+		return nil
```

Comment:
Let's print a message here on how to use `tsh aws` similar to the curl hint below.